doc/dev.md

```diff
@@ -202,6 +202,52 @@ cmake -S . -B build-x64-ios -DVCPKG_TARGET_TRIPLET=x64-ios -GXcode -DWITH_VCPKG=
 cmake --build build-x64-ios
 ```
 
+
+## iOS application (ARM-64 processors architecture)
+
+```sh
+# Firstly, some compilation dependencies need to be installed
+brew install cmake flex bison python pkg-config autoconf automake libtool
+
+# Secondly, Xcode must be installed through the AppStore, then configured (Sometimes commands like those are needed to correctly set-up Xcode)
+xcode-select --install
+sudo xcode-select --switch /Library/Developer/CommandLineTools
+sudo xcode-select -s /Applications/Xcode.app/Contents/Developer
+
+# Then, of course, Qt is needed (Could be installed with 'aqtinstall' or with the official tools)
+pip3 install aqtinstall && aqt install-qt mac ios 5.15.2 -O qt5 -m qtcharts # Qt
+
+# Then, really important, some variables need to be exported in the shell
+
+export PATH="$(brew --prefix flex)/bin:$PATH" && export PATH="$(brew --prefix bison)/bin:$PATH" # For QGis
+
+export Qt5_DIR=$HOME/Documents/qt5/5.15.2/ios/ # Seems that a full path might be needed for dependencies based on Qt like Qca
```

Comment:
Can we

- remove the `$HOME` part in here? Especially since the install command on line 218 is used with a relative path. If it requires a full path, we can just use `$(pwd)/qt5/5.15.2` instead
- remove the comment -- or at least the "Seems" and "might be" parts of it?